wolfcrypt/src/ecc.c

```diff
@@ -13588,9 +13588,12 @@ static int wc_ecc_export_x963_compressed(ecc_key* key, byte* out, word32* outLen
 
    if (*outLen < (1 + numlen)) {
       *outLen = 1 + numlen;
-      return BUFFER_E;
+      return LENGTH_ONLY_E;
    }
 
+   if (out == NULL)
+       return BAD_FUNC_ARG;
```

Comment:
Why is this split out from 13577?

Reply:
Split out to allow for `wc_ecc_export_x963_compressed` to return the length necessary without needing to provide an output buffer.